packages/docs/source/building-blocks/elements.md

```diff
@@ -0,0 +1,222 @@
+# Elements
+
+Odyssey takes care to provide additional style to the following HTML elements by default.
+
+
+
+
+## blockquote <a name="blockquote"></a>
+> The HTML `<blockquote>` Element (or HTML Block Quotation Element) indicates that the enclosed text is an extended quotation. Usually, this is rendered visually by indentation (see Notes for how to change it). A URL for the source of the quotation may be given using the cite attribute, while a text representation of the source can be given using the `<cite>` element.
+>  - <cite><a href='https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote'>MDN</a></cite>
```

Comment:
We may need to do these in HTML so that markdown parses it correctly, otherwise:

<img width="574" alt="Screenshot 2020-03-23 13 47 50" src="https://user-images.githubusercontent.com/36284167/77361665-ef4c1f80-6d0c-11ea-954c-ca723e4920f2.png">